standalone/replay/src/replay_gk.rs

```diff
@@ -343,7 +344,12 @@ fn get_checkpoint_path(from: &Option<String>) -> Option<String> {
     match from {
         Some(filename) => {
             if !filename.is_empty() {
-                Some(filename.clone())
+                if std::path::PathBuf::from(filename).exists() {
```

Comment:
Since the filename is given explictly, I think reject on not found is reasonable.